src/App2.js

```diff
@@ -0,0 +1,29 @@
+import logo from './logo.svg';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/shekhar-rudder/code-code/pullRequest?prid=7908772)
```suggestion
import logo from "./logo.svg";
```